runtime/tutor

```diff
@@ -96,7 +96,7 @@ _________________________________________________________________
  2. Move to a place in the line which is missing text and type
     i to enter Insert mode. Keys you type will now type text.
  3. Enter the missing text.
- 4. Type Escape to exit Insert mode and return to Normal mode.
+ 4. type Escape to exit Insert mode and return to Normal mode.
```

Comment:
I'm just a random person but I'm not sure why this has been lowercased? Seems like all the points start with a capital?

Reply:
Oh I didn't notice this!